python/paddle/fluid/tests/unittests/test_lamb_op.py

```diff
@@ -202,11 +219,13 @@ def update_param():
         update_mom(row_id, update_value)
 
     update_param()
+    beta1_pow_out = beta1_pow * beta1
+    beta2_pow_out = beta2_pow * beta2
 
-    return param_out, moment1_out, moment2_out
+    return param_out, moment1_out, moment2_out, beta1_pow_out, beta2_pow_out
 
 
-class TestSparseLambOp(unittest.TestCase):
+class gTestSparseLambOp(unittest.TestCase):
```

Comment:
Please remove meaningless `g` in this class name

Reply:
Thanks, done.